### ISSUE TITLE
Error: Token already used

### ISSUE THREAD

**Issue by zuphilip** [substance/texture#438]
If you open the browser toolbar at the current preview website , you see the following errors:
```
Token contrib-group already used. Ignoring this transition.		DFABuilder.js:276:4
Token abstract already used. Ignoring this transition.		DFABuilder.js:276:4
```

The same errors show up during `npm start`.

**Comment by michael** [comment]
Thanks. It's a known issue but does not affect the functionality. We'll look into fixing that.

**Comment by oliver----** [comment]
Fixed via #784
